SPECS/ksh/ksh.spec

```diff
@@ -1,51 +1,52 @@
-%global       verBetaPrefix 1.0.0
-%global       verBetaSuffix 1
-%global       verBetaFull %{verBetaPrefix}-beta.%{verBetaSuffix}
-
+%global debug_package %{nil}
 Summary:        The Original ATT Korn Shell
 Name:           ksh
-Version:        %{verBetaPrefix}~beta.%{verBetaSuffix}
-Release:        5%{?dist}
+Version:        1.0.8
+Release:        1%{?dist}
 License:        EPL-1.0
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            http://www.kornshell.com/
-Source0:        https://github.com/ksh93/%{name}/archive/v%{verBetaFull}/%{name}-%{verBetaFull}.tar.gz
+Source0:        https://github.com/ksh93/%{name}/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
 Source1:        kshcomp.conf
 Source2:        kshrc.rhs
 Source3:        dotkshrc
-# temporary commenting out failing tests
-Patch1:         %{name}-%{verBetaFull}-regre-tests.patch
-# in some build commands relocate "-lm" flag
-Patch2:         %{name}-%{verBetaFull}-fix-build.patch
 
 BuildRequires:  bison
 BuildRequires:  gcc
 # regression test suite uses 'ps' from procps
 BuildRequires:  procps
 
+# regression test suite requirements
+BuildRequires: glibc-lang
+BuildRequires: ncurses
+BuildRequires: procps
+BuildRequires: tzdata
+BuildRequires: util-linux
```

Comment:
should we move these in the below block which gates the `check` dependencies :)

Reply:
I removed them fully, we don't need them at all